### PR TITLE
Further README updates.

### PR DESCRIPTION
**Pull request by mikepurvis**
Some additional getting started notes and clarifications of the differences in approach between this project and nix-ros-overlay.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,22 +12,65 @@ by the generated snapshot flakes. However, the `flake.lock` in this repo
...
 
-To avoid rebuilding, one can optionally use the public binary cache for
-[nix-ros][nix-ros-cachix] on cachix.
+And finally, we suggest [setting up Cachix][nix-ros-cachix] so that you can
+pull pre-built binaries rather than building everything locally:
+```
+# Enter a temporary shell where cachix is available.
+nix shell github:cachix/cachix
+cachix use nix-ros
+exit
```
Comment (iwanders) [inline README.md:62]: Oh, this is nice, that way you don't end up with this persistently living in your store.

File: README.md [hunk 2]
```diff
@@ -121,21 +201,40 @@ arguments is invoked. This provides developers with sane defaults that are
...
 these were prepared specifically for a talk given at ROSCon 2022.
 
+[nixros]: https://github.com/clearpathrobotics/nix-ros/
+[gc]: https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-store-gc.html
+
+
 ## Credits
 
-This work would not have been possible without @lopsided98 's [nix-ros-overlay](https://github.com/lopsided98/nix-ros-overlay/) repository.
+This work would not have been possible without @lopsided98's work on packaging
```
Comment (iwanders) [inline README.md]: I think we should just link to @lopsided98's github profile here, usernames don't become clickable links in the readme with `@`.

File: README.md [hunk 3]
```diff
@@ -12,22 +12,65 @@ by the generated snapshot flakes. However, the `flake.lock` in this repo
 is critically important, as it controls which versions of our upstream
 dependencies (nixpkgs, nix-ros-overlay) we are pinned to.
 
-## Usage
+## Nix Installation
```
Comment (iwanders) [inline README.md:15]: I'd personally shy away from this. This will always go stale, I'd link to the upstream documentation only.

**Review by iwanders — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mikepurvis) [reply README.md:62]: I think you could even just do a one-off `nix run` but it's nice to demonstrate nix's ephemeral subshell workflow here.

File: README.md (hunk 2 above)
Comment (mikepurvis) [reply README.md]: Yes good call, will fix.

File: README.md (hunk 3 above)
Comment (mikepurvis) [reply README.md:15]: I'm of two minds. Yes, generally I'd agree, but given that this repo is specifically targeting nix novices, I don't want to send them on a wild goose chase of external links and wrangling configuration.

File: README.md (hunk 2 above)
Comment (mikepurvis) [reply README.md]: And for @lopsided98, no rush, but when you have time, we'd be delighted for you to look over this comparison to ensure it is fair and accurate. We're more than happy to accept corrections.

Comment (lopsided98) [reply README.md]: It looks fine to me.

**Pull request merged**